backends/ebpf/codeGen.cpp

```diff
@@ -242,7 +242,7 @@ bool CodeGenInspector::preorder(const IR::AssignmentStatement* a) {
 bool CodeGenInspector::preorder(const IR::BlockStatement* s) {
     builder->blockStart();
     setVecSep("\n", "\n");
-    visit(s->components);
+    s->components.visit_children(*this);
```

Comment:
Well, ideally, the code should be using the default visit_children code that is built by the IR generator rather than unnecessarily using custom visit orders in the preorder function.